#! /bin/sh /usr/share/dpatch/dpatch-run
## 14_charsign_fixes.dpatch by  <till.kamppeter@gmail.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: some missing charsign

@DPATCH@
diff -urNad hplip-1.7.3~/scan/sane/scl.c hplip-1.7.3/scan/sane/scl.c
--- hplip-1.7.3~/scan/sane/scl.c	2007-03-19 16:26:48.000000000 +0000
+++ hplip-1.7.3/scan/sane/scl.c	2007-03-26 22:48:39.000000000 +0100
@@ -36,10 +36,11 @@
 #define DEBUG_DECLARE_ONLY
 #include "sanei_debug.h"
 
-static int SclBufferIsPartialReply( unsigned char * data, int datalen )
+static int SclBufferIsPartialReply( void * dataptr, int datalen )
 {
     int i = 0, value = 0;
-    unsigned char d;  
+    unsigned char * const data = dataptr;
+    unsigned int d;
 
     if( i >= datalen )
     {
